src/iidm/converter/ExportOptions.cpp

```diff
@@ -21,26 +21,51 @@ namespace powsybl {
 
 namespace iidm {
 
+namespace Enum {
+
+template <>
+const std::initializer_list<std::string>& getNames<converter::ExportOptions::IidmVersionIncompatibilityBehavior>() {
+    static std::initializer_list<std::string> s_names {
+        "THROW_EXCEPTION",
+        "LOG_ERROR"
+    };
+    return s_names;
+}
+
+}  // namespace Enum
+
 namespace converter {
 
 static const Parameter ANONYMISED_PARAMETER(ExportOptions::ANONYMISED, Parameter::Type::BOOLEAN, "Anonymize exported network", "false");
 static const Parameter EXTENSIONS_LIST_PARAMETER(ExportOptions::EXTENSIONS_LIST, Parameter::Type::STRING_LIST, "The list of exported extensions", "");
+static const Parameter IIDM_VERSION_INCOMPATIBILITY_BEHAVIOR_PARAMETER (ExportOptions::IIDM_VERSION_INCOMPATIBILITY_BEHAVIOR, Parameter::Type::STRING, "Behavior when there is an IIDM version incompatibility", "THROW_EXCEPTION");
 static const Parameter INDENT_PARAMETER(ExportOptions::INDENT, Parameter::Type::BOOLEAN, "Indent export output file", "true");
 static const Parameter ONLY_MAIN_CC_PARAMETER(ExportOptions::ONLY_MAIN_CC, Parameter::Type::BOOLEAN, "Export only main CC", "false");
-static const Parameter THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_PARAMETER = Parameter(ExportOptions::THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND, Parameter::Type::BOOLEAN, "Throw exception if extension not found", "false")
-    .addAdditionalNames({"throwExceptionIfExtensionNotFound"});
+static const Parameter THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_PARAMETER = Parameter(ExportOptions::THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND, Parameter::Type::BOOLEAN, "Throw exception if extension not found", "false").addAdditionalNames({"throwExceptionIfExtensionNotFound"});
 static const Parameter TOPOLOGY_LEVEL_PARAMETER(ExportOptions::TOPOLOGY_LEVEL, Parameter::Type::STRING, "Export network in this topology level", "NODE_BREAKER");
 static const Parameter VERSION_PARAMETER(ExportOptions::VERSION, Parameter::Type::STRING, "IIDM-XML version in which files will be generated", xml::IidmXmlVersion::CURRENT_IIDM_XML_VERSION().toString("."));
 static const Parameter WITH_BRANCH_STATE_VARIABLES_PARAMETER(ExportOptions::WITH_BRANCH_STATE_VARIABLES, Parameter::Type::BOOLEAN, "Export network with branch state variables", "true");
 
+std::ostream& operator<<(std::ostream& stream, const ExportOptions::IidmVersionIncompatibilityBehavior& value) {
+    stream << iidm::Enum::toString(value);
+    return stream;
+}
+
 ExportOptions::ExportOptions(bool withBranchSV, bool indent, bool onlyMainCc, const TopologyLevel& topologyLevel,
                              bool throwExceptionIfExtensionNotFound, const std::string& version) :
+    ExportOptions(withBranchSV, indent, onlyMainCc, topologyLevel, throwExceptionIfExtensionNotFound, version, IidmVersionIncompatibilityBehavior::THROW_EXCEPTION) {
+}
```

Comment:
It's not necessary to keep both constructors in C++. It's done like this in Java to ensure compatibility but we don't care here